packages/app/src/cli/models/extensions/specifications/flow_action.ts

```diff
@@ -12,6 +13,7 @@ import {zod} from '@shopify/cli-kit/node/schema'
 
 export const FlowActionExtensionSchema = BaseSchema.extend({
   name: zod.string(),
+  handle: zod.string().refine(validateHandle),
```

Comment:
Shouldn't the handle be on the `extension` directly? We can double check with @isaacroldan but I believe that's where the CLI is heading since there could be more than one extension per file.

Reply:
I believe you're right, will update. The specification type is in both places so I wasn't sure, but the prototype code has it inside the extensions array.